File: AlgoExpert/Greedy Algorithms/Easy/Tandem Bicycle.java
import java.util.*;

class Program {

  public int tandemBicycle(int[] redShirtSpeeds, int[] blueShirtSpeeds, boolean fastest) {
    // Write your code here.
		Arrays.sort(redShirtSpeeds);
		Arrays.sort(blueShirtSpeeds);
		int maxTotalSpeed = 0;
		
		for (int i = 0; i < redShirtSpeeds.length; i++) {
			int j = fastest ? redShirtSpeeds.length - i - 1 : i;
			maxTotalSpeed += Math.max(redShirtSpeeds[i], blueShirtSpeeds[j]);
		}
		
    return maxTotalSpeed;
  }
}

File: Python/AlgoExpert/Greedy Algorithms/Easy/tandemBicycle.py
def tandemBicycle(redShirtSpeeds, blueShirtSpeeds, fastest):
    # Write your code here.
    redShirtSpeeds.sort()
    blueShirtSpeeds.sort(reverse=fastest)

    speed_sums = 0
    for i in range(len(redShirtSpeeds)):
        speed_sums += max(redShirtSpeeds[i], blueShirtSpeeds[i])
    
    return speed_sums
